src/components/modules/toolbar/blockSettings.ts

```diff
@@ -197,6 +197,100 @@ export default class BlockSettings extends Module<BlockSettingsNodes> {
     }
   };
 
+  /**
+   * Returns list of items to be displayed in block tunes menu.
+   * Merges tool specific tunes, conversion menu and common tunes in one list in predefined order
+   *
+   * @param toolName - current block tool name
+   * @param commonTunes – common tunes
+   * @param toolTunes - tool specific tunes
+   */
+  private getTunesItems(toolName: string, commonTunes: TunesMenuConfigItem[], toolTunes?: TunesMenuConfigItem[]): PopoverItemParams[] {
+    const items = [] as TunesMenuConfigItem[];
+
+    if (toolTunes !== undefined && toolTunes.length > 0) {
+      items.push(...toolTunes);
+      items.push({
+        type: 'separator',
+      });
+    }
+
+    /**
+     * Exclude current tool from "convert to" items list
+     */
+    const convertToItems = this.allConvertToItems.filter(item => item.name !== toolName);
+
+    if (convertToItems.length > 0) {
+      items.push({
+        type: 'default',
+        icon: IconReplace,
+        title: I18n.ui(I18nInternalNS.ui.popover, 'Convert to'),
+        children: {
+          items: convertToItems,
+        },
+      });
+      items.push({
+        type: 'separator',
+      });
+    }
+
+    items.push(...commonTunes);
+
+    return items.map(tune => this.resolveTuneAliases(tune));
+  }
+
+  /**
+   * List of all available conversion menu items
+   */
+  @_.cacheable
+  private get allConvertToItems(): PopoverItemDefaultParams[] {
+    const conversionEntries = Array.from(this.Editor.Tools.blockTools.entries());
+
+    const resultItems: PopoverItemDefaultParams[] = [];
+
+    conversionEntries.forEach(([toolName, tool]) => {
+      const conversionConfig = tool.conversionConfig;
+
+      /**
+       * Skip tools without «import» rule specified
+       */
+      if (!conversionConfig || !conversionConfig.import) {
+        return;
+      }
```

Comment:
can we use existing `isBlockConvertable()` here?

Reply:
I don't know, I copied the logic from module [conversion.ts](https://github.com/codex-team/editor.js/blob/next/src/components/modules/toolbar/conversion.ts#L213)

Reply:
You should at least try. This method added recently. All such statements should be utilized

Reply:
at this phase we do not have a block to check if it is convertable or not. We operate with tools, not blocks

Reply:
you have this.Editor.Tools.blockTools above

Reply:
Those are block tools, not blocks
![image](https://github.com/codex-team/editor.js/assets/31101125/b4f365d8-bc34-4f1b-a71c-13bd6f6f186e)